src/datadog/azext_datadog/generated/_params.py

```diff
@@ -121,41 +127,48 @@ def load_arguments(self, _):
         c.argument('resource_group_name', resource_group_name_type)
         c.argument('monitor_name', type=str, help='Monitor resource name')
         c.argument('rule_set_name', type=str, help='Rule set name')
-        c.argument('metric_rules_filtering_tags', action=AddMetricRulesFilteringTags, nargs='*', help='List of '
-                   'filtering tags to be used for capturing metrics. If empty, all resources will be captured. If only '
-                   'Exclude action is specified, the rules will apply to the list of all available resources. If '
-                   'Include actions are specified, the rules will only include resources with the associated tags.')
-        c.argument('log_rules_send_aad_logs', arg_type=get_three_state_flag(), help='Flag specifying if AAD logs '
-                   'should be sent for the Monitor resource.')
-        c.argument('log_rules_send_subscription_logs', arg_type=get_three_state_flag(), help='Flag specifying if Azure '
-                   'subscription logs should be sent for the Monitor resource.')
-        c.argument('log_rules_send_resource_logs', arg_type=get_three_state_flag(), help='Flag specifying if Azure '
-                   'resource logs should be sent for the Monitor resource.')
-        c.argument('log_rules_filtering_tags', action=AddLogRulesFilteringTags, nargs='*', help='List of filtering '
+        c.argument('provisioning_state', arg_type=get_enum_type(['Accepted', 'Creating', 'Updating', 'Deleting',
+                                                                 'Succeeded', 'Failed', 'Canceled', 'Deleted',
+                                                                 'NotSpecified']), help='Provisioning state')
```

Comment:
`provisioning_state` should be readonly. Please fix the swagger definition and regenerate from the fixed swagger.